src/Tests/GlobalRedirectTest.php

```diff
@@ -41,9 +41,10 @@ class GlobalRedirectTest extends WebTestBase {
    */
   protected $config;
 
-  protected $forumTerm;
-  protected $term;
-  protected $node;
+  /**
+   * @var \Drupal\Core\Entity\ContentEntityInterface
+   */
+  protected $forumTerm, $term, $node;
```

Comment:
I don't think our coding standard allows that. Can you keep the separate and add the @ var everywhere?